db_stress_tool/db_stress_gflags.cc

```diff
@@ -450,6 +450,24 @@ DEFINE_int32(
     "[Integrated BlobDB] Enable writing blob files during flushes and "
     "compactions starting from the specified level.");
 
+DEFINE_bool(use_blob_cache, false, "[Integrated BlobDB] Enable blob cache.");
+
+DEFINE_bool(
+    use_shared_block_and_blob_cache, true,
+    "[Integrated BlobDB] Use a shared backing cache for both block "
+    "cache and blob cache. It only takes effect if use_blob_cache is enabled.");
+
+DEFINE_uint64(
+    blob_cache_size, FLAGS_cache_size,
```

Comment:
Is it safe to use the value of a different option as a default like this? If not, we might want to go with hardcoded defaults (also applies to shard bits below)

Reply:
Make sense. Done